core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -81,6 +81,10 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
   private val SPARK_HISTORY_FS_NUM_REPLAY_THREADS = "spark.history.fs.numReplayThreads"
 
+  private val AppStartEvent = "{\"Event\":\"SparkListenerApplicationStart\""
```

Comment:
These should really be in a companion object anyway, including existing constants.